types/parcel-bundler/index.d.ts

```diff
@@ -173,6 +179,12 @@ declare class ParcelBundler {
     addPackager(type: string, packager: string): void;
 
     bundle(): Promise<ParcelBundler.ParcelBundle>;
+
+    serve(
+        port?: number,
+        https?: boolean | { cert: string; key: string },
+        host?: string
+    ): http.Server | https.Server;
```

Comment:
[the method is asynchronous](https://github.com/parcel-bundler/parcel/blob/6cbf2a65cdcd66af135ff9eca2a3799bde9f8a5b/packages/core/parcel-bundler/src/Bundler.js#L787)

```js
async serve(port = 1234, https = false, host) {
```

so, i think, it should be like that

```ts
Promise<http.Server | https.Server>;
```

Reply:
Damn, you're totally right @pinage404. Thanks!